.expeditor/config.yml

```diff
@@ -51,3 +51,10 @@ subscriptions:
       - built_in:promote_habitat_packages
       - trigger_pipeline:deploy/acceptance:
           post_commit: true
+
+  # This will update version of the Chef product Hugo module
+  # that is used to build the docs when that product is promoted to stable.
+  - workload: artifact_published:stable:chef-workstation:*
```

Comment:
You might want to limit this to current major version. We probably don't want to confuse this with documentation of previous major versions (when one is released).

Reply:
I'm not sure I understand. While we used to link chef-web-docs to release versions, we no longer do this. We're working towards making docs releases part of the normal product build process, so (If I understand correctly) every promotion to current could have new docs updates. Syncing to the previous major version would miss all of this work.